packages/venia-concept/validate-environment.js

```diff
@@ -0,0 +1,94 @@
+function validateEnvironment(env) {
+    const { readFileSync } = require('fs');
+    const path = require('path');
+    const dotenv = require('dotenv');
+    const chalk = require('chalk');
+
+    let parsedEnv;
+    const envPath = path.join(__dirname, '.env');
+    try {
+        parsedEnv = dotenv.parse(readFileSync(envPath));
```

Comment:
It a good practice to use also encoding for parsing `.env` files, because it can be an error for people who used another encoding
``` js
parsedEnv = dotenv.parse({path: readFileSync(envPath), encoding: 'utf-8'});
```

Reply:
thanks @vitalics !
In that scenario, it won't be applied even being `utf8` the [default encoding](https://www.npmjs.com/package/dotenv#encoding)? (I'm just curious...)
maybe will be better to specify that at `require().config(xxxx)` level ?
cause we're using the `.env` in a lot of places, maybe it's a good chance to standardize this.

If that's the case, regarding `utf-8` value:
would make sense to specify as `utf8` and be aligned on how [the lib specifies this param](https://github.com/clakech/dotenv/blob/master/lib/main.js#L48) in their docs?
(despite that `readFileSync()` and `Buffer` from node _understands_ both values).

what do you think?

Reply:
@mhhansen 
Yes, by default it's  `utf-8` encoding, but what about the cross various solution. Read more about this on [this ](http://unicode.org/faq/utf_bom.html) article in next section: `Which of the UTFs do I need to support?`.
Quote: `UTF-8 is most common on the web. UTF-16 is used by Java and Windows. UTF-8 and UTF-32 are used by Linux and various Unix systems.`

I agree with you that we can make it on `require` level for defining it visually, it will be more clarify and understandable for others persons on this project